platform_versioned_docs/version-24.1/cloud/changelog.mdx

```diff
@@ -6,9 +6,21 @@ tags: [changelog]
 ---
 ## 2024
 
+### 24.2.0.cycle21 - 14 August 2024
+
+- Feature: Add global Nextflow configs in compute environments
+- Feature: Added flexibility for pipeline names in workspaces
+- Feature: Added tags propagation to launch template
+- Feature: Add backend and UI to include managed identities into the configuration when using a manual Azure CE
+- Feature: Implement custom launch container logic
+- Fix: Improve workflow launch L&F
```

Comment:
What is L&F?

Reply:
Taking it out for now as the source PR is unclear.